habitat/plan.sh

```diff
@@ -136,21 +145,26 @@ do_strip() {
 
 # Copied from https://github.com/habitat-sh/core-plans/blob/f84832de42b300a64f1b38c54d659c4f6d303c95/bundler/plan.sh#L32
 wrap_ruby_bin() {
-  local bin_basename="$1"
-  local real_cmd="$pkg_prefix/ruby-bin/$bin_basename"
-  local wrapper="$pkg_prefix/bin/$bin_basename"
-
-  build_line "Adding wrapper $wrapper to $real_cmd"
+  local bin_basename
+  local real_cmd
+  local wrapper
+  bin_basename="$1"
+  real_cmd="$ruby_bin_dir/$bin_basename"
+  wrapper="$pkg_prefix/bin/$bin_basename"
+
+  build_line "Adding wrapper for $bin_basename."
+  build_line " - from: $wrapper"
+  build_line " -   to: $real_cmd"
   cat <<EOF > "$wrapper"
 #!$(pkg_path_for busybox-static)/bin/sh
 set -e
-if test -n "$DEBUG"; then set -x; fi
+if test -n "\$DEBUG"; then set -x; fi
```

Comment:
Before we were actually printing the content of the variable, now we are writing the variable name to check it at execution of these binaries, you can now do this:
```
[86][default:/src:1]# DEBUG=true chef
+ export 'GEM_HOME=/hab/pkgs/afiune/chef-dk/4.3.3/20190730094053/ruby/2.6.0/'
+ export 'GEM_PATH=/hab/pkgs/core/ruby26/2.6.3/20190515141800/lib/ruby/gems/2.6.0:/hab/pkgs/core/bundler/1.17.3/20190416230243:/hab/pkgs/afiune/chef-dk/4.3.3/20190730094053/ruby/2.6.0/:/hab/pkgs/afiune/chef-dk/4.3.3/20190730094053'
+ export 'SSL_CERT_FILE=/hab/pkgs/core/cacerts/2018.12.05/20190115014206/ssl/cert.pem'
+ export 'APPBUNDLER_ALLOW_RVM=true'
+ unset RUBYOPT GEMRC
+ exec /hab/pkgs/core/ruby26/2.6.3/20190515141800/bin/ruby /hab/pkgs/afiune/chef-dk/4.3.3/20190730094053/ruby-bin/chef
/hab/pkgs/core/ruby26/2.6.3/20190515141800/bin/ruby
Usage:
    chef -h/--help
    chef -v/--version
    chef command [arguments...] [options...]


Available Commands:
    exec                    Runs the command in context of the embedded ruby
    env                     Prints environment variables used by ChefDK
    gem                     Runs the `gem` command in context of the embedded Ruby
    generate                Generate a new repository, cookbook, or other component
    shell-init              Initialize your shell to use ChefDK as your primary Ruby
    install                 Install cookbooks from a Policyfile and generate a locked cookbook set
    update                  Updates a Policyfile.lock.json with latest run_list and cookbooks
    push                    Push a local policy lock to a policy group on the Chef Infra Server
    push-archive            Push a policy archive to a policy group on the Chef Infra Server
    show-policy             Show policyfile objects on the Chef Infra Server
    diff                    Generate an itemized diff of two Policyfile lock documents
    export                  Export a policy lock as a Chef Infra Zero code repo
    clean-policy-revisions  Delete unused policy revisions on the Chef Infra Server
    clean-policy-cookbooks  Delete unused policyfile cookbooks on the Chef Infra Server
    delete-policy-group     Delete a policy group on the Chef Infra Server
    delete-policy           Delete all revisions of a policy on the Chef Infra Server
    undelete                Undo a delete command
    describe-cookbook       Prints cookbook checksum information used for cookbook identifier
```